frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/components/Tabs/Maskinporten/Maskinporten.tsx

```diff
@@ -1,17 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classes from './Maskinporten.module.css';
 import { useTranslation } from 'react-i18next';
 import { TabContent } from '../../TabContent';
-import { StudioButton, StudioHeading, StudioParagraph, StudioSpinner } from '@studio/components';
+import {
+  StudioAlert,
+  StudioButton,
+  StudioHeading,
+  StudioParagraph,
+  StudioSpinner,
+} from '@studio/components';
 import { useIsLoggedInWithAnsattportenQuery } from '../../../../../../../../hooks/queries/useIsLoggedInWithAnsattportenQuery';
 
 export const Maskinporten = (): React.ReactElement => {
+  const [scopes, setScopes] = useState(undefined);
```

Comment:
I think we should make a new `useGetScopesQuery()` hook with tanstack query that fetches the scopes. That way we do not need to have useState for scopes. 😊 Just, mock the response for now, until the backend is ready. 😊

Reply:
Done 😄